### DISCUSSION TITLE
Collecting Crash Metrics using Sentry.io

### DISCUSSION THREAD

**Discussion by arky** [aurelienpierreeng/ansel#795] [Ideas]
Perhaps it would be good to use some kind of metrics system to collect data about crashes using tools like Sentry.io or something similar. 

AFAIK it should be possible to hook up tools to report data and sentry does provide a free tier for open source projects. 

**Additional context**

https://sentry.io/for/open-source/
https://sentry.io/for/c-plus-plus/

**Comment by aurelienpierre** [comment]
Maybe one day®

**Comment by pedrorrodriguez** [comment]
Discussions?

**Comment by aurelienpierre** [comment]
2 Problems here :

1. who will read/sort/triage/compile the crash logs ? We don't have the workforce for that,
2. relying on another third-party service that may break API or change pricing policy anytime is a no-go for me.

**Comment by Jiyone** [comment]
I won't do after-sales service x_x

**Comment by pedrorrodriguez** [comment]
Then let's close. I just migrated the issue over to discussions so that it wouldn't clutter the tracker
